Rakefile

```diff
@@ -46,11 +46,6 @@ RDoc::Task.new :rdoc => 'docs', :clobber_rdoc => 'clobber_docs' do |doc|
   doc.rdoc_dir = 'doc'
 end
 
-desc "Install gems needed to run the tests"
-task :install_test_deps => :clean do
-  sh "gem install minitest -v '~> 5.0'"
```

Comment:
The other place where this is defined is 

https://github.com/rubygems/rubygems/blob/a0b3d351043e824819b3d66ce3e7d6958b4ba252/util/ci#L55

Right?

Maybe we can install move that line here and call the rake task from there?

Reply:
I update it for using `rake setup` in `util/ci`